### PR TITLE
Added security information to swagger

### PR DESCRIPTION
**Pull request by ruks**
Added security information to swagger

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java [hunk 1]
```diff
@@ -426,48 +427,66 @@ public static JsonNode generateOASConfigForEndpoints(API api, boolean isProducti
...
         } else if (APIConstants.ENDPOINT_TYPE_HTTP.equalsIgnoreCase(type)) {
-            setPrimaryConfig(endpointResult, endpointConfig, isProduction, APIConstants.ENDPOINT_TYPE_HTTP);
+            endpointResult = setPrimaryConfig(endpointConfig, isProduction, APIConstants.ENDPOINT_TYPE_HTTP);
         } else if (APIConstants.ENDPOINT_TYPE_ADDRESS.equalsIgnoreCase(type)) {
-            setPrimaryConfig(endpointResult, endpointConfig, isProduction, APIConstants.ENDPOINT_TYPE_ADDRESS);
+            endpointResult = setPrimaryConfig(endpointConfig, isProduction, APIConstants.ENDPOINT_TYPE_ADDRESS);
         } else {
             return null;
         }
+        if(endpointResult != null) {
```
Comment (bhathiya) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java]: need to format.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java [hunk 2]
```diff
@@ -426,48 +427,66 @@ public static JsonNode generateOASConfigForEndpoints(API api, boolean isProducti
...
+            if(endpointConfig.has(APIConstants.ENDPOINT_PRODUCTION_ENDPOINTS)) {
+                primaryEndpoints = endpointConfig.getJSONObject(APIConstants.ENDPOINT_PRODUCTION_ENDPOINTS);
+            }
         } else {
-            endpointsURLs = endpointConfig.getJSONArray(APIConstants.ENDPOINT_SANDBOX_FAILOVERS);
-            primaryEndpoints = endpointConfig.getJSONObject(APIConstants.ENDPOINT_SANDBOX_ENDPOINTS);
+            if(endpointConfig.has(APIConstants.ENDPOINT_SANDBOX_FAILOVERS)) {
+                endpointsURLs = endpointConfig.getJSONArray(APIConstants.ENDPOINT_SANDBOX_FAILOVERS);
+            }
+            if(endpointConfig.has(APIConstants.ENDPOINT_SANDBOX_ENDPOINTS)) {
```
Comment (bhathiya) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java]: Please format the code.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java [hunk 3]
```diff
@@ -426,48 +427,66 @@ public static JsonNode generateOASConfigForEndpoints(API api, boolean isProducti
...
+    private static ObjectNode populateFailoverConfig(JSONObject endpointConfig, boolean isProd) {
+        JSONArray endpointsURLs = null;
+        JSONObject primaryEndpoints = null;
         if (isProd) {
-            endpointsURLs = endpointConfig.getJSONArray(APIConstants.ENDPOINT_PRODUCTION_FAILOVERS);
-            primaryEndpoints = endpointConfig.getJSONObject(APIConstants.ENDPOINT_PRODUCTION_ENDPOINTS);
+            if(endpointConfig.has(APIConstants.ENDPOINT_PRODUCTION_FAILOVERS)) {
+                endpointsURLs = endpointConfig.getJSONArray(APIConstants.ENDPOINT_PRODUCTION_FAILOVERS);
+            }
+            if(endpointConfig.has(APIConstants.ENDPOINT_PRODUCTION_ENDPOINTS)) {
```
Comment (bhathiya) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java]: Please format the code.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java [hunk 4]
```diff
@@ -480,63 +499,80 @@ private static void populateFailoverConfig(ObjectNode endpointResult, JSONObject
         if (primaryEndpoints != null && primaryEndpoints.has(APIConstants.ENDPOINT_URL)) {
             endpointsArray.add(primaryEndpoints.getString(APIConstants.ENDPOINT_URL));
         }
+        if(endpointsArray.size() < 1) {
```
Comment (bhathiya) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java]: Please format the code.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java (hunk 1 above)
Comment (ruks) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OASParserUtil.java]: Fixed in https://github.com/wso2/carbon-apimgt/pull/7500

**Pull request merged**